.travis.yml

```diff
@@ -0,0 +1,55 @@
+language: python
+python:
+  - "2.7"
+sudo: required
+services:
+  - docker
+env:
+  global:
+    - PA_TEST_ONLINE_INSTALLER=true
+    - PYTHONPATH=$(pwd)
+  matrix:
+    - OTHER_TESTS=true
+    - PRODUCT_TEST_GROUP_SA_BARE="tests/product/test_installer.py tests/product/standalone/test_installation.py"
+    - PRODUCT_TEST_GROUP_SA_PRESTO="tests/product/test_authentication.py tests/product/test_configuration.py tests/product/test_server_upgrade.py"
+    - PRODUCT_TEST_GROUP_SA_PA="tests/product/test_error_handling.py tests/product/test_package_install.py tests/product/test_plugin.py tests/product/test_script.py tests/product/test_topology.py"
+    - PRODUCT_TEST_GROUP_SA_PA="tests/product/test_status.py"
+    - PRODUCT_TEST_GROUP_SA_PA="tests/product/test_server_install.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_collect.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_connectors.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_control.py"
+    - PRODUCT_TEST_GROUP_SA_PA_PRESTO="tests/product/test_server_uninstall.py"
+    - PRODUCT_TEST_GROUP_YS_PA="tests/product/yarn_slider/test_slider_installation.py"
+    #-QUARANTINED="tests/product/yarn_slider/test_server_install.py tests/product/test_create_images.py"
```

Comment:
I can uncomment it. They're the quarantined tests, so they don't end up running but would trigger another job for travis to do.